### DISCUSSION TITLE
使用apache-iotdb的python包查询效率低

### DISCUSSION THREAD

**Discussion by tkatt** [apache/iotdb#4238] [Q&A]
使用官方提供的apache-iotdb的python包，执行sql查询结果大约400万条记录，将结果解析到python的list的过程非常慢。耗时大约5分钟。
相同的sql，java只需要15s

**Comment by HTHou** [comment]
- https://github.com/apache/iotdb/pull/4242 

有一个PR提升了python client 查询 todf 方法的效率，0.13 版本发布后可以体验一下
